contracts/lockdrop-pcl/src/contract.rs

```diff
@@ -658,6 +664,7 @@ pub fn callback_withdraw_user_rewards_for_lockup_optional_withdraw(
     user_address: Addr,
     duration: u64,
     withdraw_lp_stake: bool,
+    reward_tokens: Vec<AssetInfo>,
```

Comment:
There is a detailed comment describing parameters just above this function. It would make sense to add `reward_tokens` there, too.